install.sh

```diff
@@ -323,8 +305,30 @@ function extract_install () {
       fi
     fi
 
-    echo_intra "Installing ${1} ..."
+    echo_intra "Installing ${1}..."
     tar cpf - ./*/* | (cd /; tar xp --keep-directory-symlink -m -f -)
+
+    if [[ "${1}" == 'glibc_standalone' ]]; then
+      # set LD_AUDIT to ignore libC.so.6 requests
+      export LD_AUDIT="${CREW_PREFIX}/opt/glibc-libs/crew-audit.so"
+
+      # setup ld.so.conf and update cache
+      cp /etc/ld.so.conf "${CREW_PREFIX}/etc/ld.so.conf"
+      echo "${CREW_PREFIX}/opt/glibc-libs" >> "${CREW_PREFIX}/etc/ld.so.conf"
+      ldconfig
+    else
+      # decompress and switch to glibc_standalone for existing binaries
+      if command -v upx &> /dev/null; then
+        echo_intra "Running upx on ${1}..."
+        grep "/usr/local/\(bin\|lib\|lib${LIB_SUFFIX}\)" < filelist | xargs -P "$(nproc)" -n1 upx -d 2> /dev/null || true
```

Comment:
This will probably be less verbose:
```bash
grep "/usr/local/\(bin\|lib\|lib${LIB_SUFFIX}\)" < filelist | xargs -P "$(nproc)" -n1 upx -qq -d 2> /dev/null || true
```

Reply:
Change made